src/controllers/userController.js

```diff
@@ -219,6 +217,58 @@ const socialAuth = async (req, res) => {
     return response(res, 500, 'error', { message: serverError });
   }
 };
+/**
+  * Handles user reset password
+  *
+  * @function
+  * @param {Object} req - request object to the server
+  * @param {Object} res - response object from the server
+  *
+  * @returns {Object} - password reset link
+*/
+const forgotPassword = async (req, res) => {
+  try {
+    const { email } = req.body;
+    const user = await findByEmail(email);
+    if (!user) {
+      const token = generateToken(email, '10m');
+      const link = `${process.env.FONTEND_BASE_URL}/reset/password/1/${token}`;
+      const message = createTemplate(resetPasswordMessage, link);
+      await sendMail(email, 'Reset Password', message);
```

Comment:
There no need to send an email if user dont exist just send a success response only

Reply:
changed to just success response